doc/requirements/io_class

```diff
@@ -0,0 +1,78 @@
+---
+group: IO class
+---
+
+Requirements for IO classes
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting occupancy limit
+id: occupancy
+---
+
+It shall be possible to set the maximum size of an IO class.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed occupancy values
+id: occupancy_allowed_values
+---
+
+The product shall accept integer and fixed point values for a max size of an IO class
+in range from 0.00 to 1.00 and interpret them as a fraction of a cache size.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Default occupancy limit
+id: occupancy_default_limit
+---
+
+When initializing a cache the software shall set IO class's max occupancy to 1.00.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting an occupancy limit on loading a cache
+id: occupancy_limit_on_load
+---
+
+When a cache state is loaded, the occupancy limit for each IO class shall be set
+to the value used before the cache had been shut down.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Display an occupancy limit value
+id: occupancy_display_limit
+---
+
+It shall be possible to display the currently set value of the occupancy limit
+for each configured ioclass. The value shall be displayed as a fraction of a cache
+size.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Respecting occupancy limit on a cache miss
+id: occupancy_respecting_limit_on_miss
+---
+
+When an I\O request is a miss and the target IO class has reached its occupancy limit,
+the software shall evict a missing number of cache blocks from the target IO class
+before inserting new data into it.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: I\O request to a disabled IO class
+id: occupancy_disabled_io_class
+---
+
+When a target IO class max occupancy has been set to 0.00 the software shall submit
+I\O requests in the Pass-Through mode.
```

Comment:
Why such an inconsistent spelling of I/O? (I\O request vs IO class??)

Reply:
"I\O" looks really strange with a backslash. "I/O" is definitely more common:
https://en.wikipedia.org/wiki/Input/output